src/main/java/io/excaliburfrc/robot/subsystems/LEDS.java

```diff
@@ -0,0 +1,34 @@
+package io.excaliburfrc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import io.excaliburfrc.robot.Constants.LedsConstants;
+
+public class LEDS extends SubsystemBase {
+  private final PWM leds = new PWM(LedsConstants.LEDS_PORT);
+
+  public enum LedMode {
+    BLUE(0.87),
+    RED(0.61),
+    GREEN(0.73),
+    YELLOW(0.67),
+    RAINBOW(-0.97),
```

Comment:
can you add white as a color? we will probably want as bright as we can for the line detection

Reply:
I can't find the white color code...

Reply:
Per the [user manual](https://www.revrobotics.com/content/docs/REV-11-1105-UM.pdf), white is `0.93`.